tests/unit/LoggerTests.cpp

```diff
@@ -179,7 +179,7 @@ TEST_F(LoggerInitTest, ChannelLogLevel)
 
 TEST_F(LoggerInitTest, InitReturnsErrorIfCouldNotCreateLogDirectory)
 {
-    auto const parsingErrors = config_.parse(ConfigFileJson{boost::json::object{{"log_directory", "/root"}}});
+    auto const parsingErrors = config_.parse(ConfigFileJson{boost::json::object{{"log_directory", "/proc/logs"}}});
```

Comment:
Worth adding a comment why `/proc/logs` was chosen. Something about it being always readonly i guess.